bindings_ffi/src/mls.rs

```diff
@@ -144,6 +168,25 @@ impl FfiXmtpClient {
     pub fn installation_id(&self) -> Vec<u8> {
         self.inner_client.installation_public_key()
     }
+
+    pub async fn release_db_connection(&self) -> Result<(), GenericError> {
+        if let Some(store) = STORE_MANAGER.lock().unwrap().get_store() {
+            store.lock().unwrap().release_connection()?;
+        }
+
+        Ok(())
+    }
```

Comment:
Should I set a flag in here like `needs_to_be_reconnected?` and then check that through out the code and reconnect only if that is set? Or do you think it would be sufficient to tell clients to call reconnect themselves?